src/main/resources/assets/bloodmagic/lang/ru_ru.json

```diff
@@ -0,0 +1,504 @@
+{
+  "guide.bloodmagic.name": "§4§lКровавый Том",
+  "guide.bloodmagic.subtitle": "§9§oМагия Алхимии", 
+  "guide.bloodmagic.landing": "Добро пожаловать в $(6)Blood Magic$()! $(br2)$(l:bloodmagic:utility/nyi)Многие вещи$() ещё не реализованы, так что прошу прощения. $(br2)Нажмите $(l:bloodmagic:utility/getting_started)СЮДА$() чтобы начать. Если вы обнаружите какие-либо ошибки, сообщите о них на нашем $(l:https://github.com/WayofTime/BloodMagic/issues)Github$().",
```

Comment:
The changes currently implemented for the next update [includes 4 more lines to be translated](https://github.com/WayofTime/BloodMagic/blob/f744078f63db5f5b20a08a92a9551db1302b627f/src/generated/resources/assets/bloodmagic/lang/en_us.json#L249-L252).

Possibly tweaking resource names was mentioned in an en_us comment.
 
```
  "patchouli.bloodmagic.arc_processor.fluid": "%dmb of %s",
  "patchouli.bloodmagic.arc_processor.no_fluid": "None",
  "patchouli.bloodmagic.living_armour_upgrade_table.level": "Level",
  "patchouli.bloodmagic.living_armour_upgrade_table.upgrade_points": "Upgrade Points",
```

for `"%dmb of %s"`, only the "mb of" (millibuckets of) would need to be translated.  `%d` and `%s` are formatting codes.